Sources/Brave/WebFilters/BraveS3Resource.swift

```diff
@@ -12,6 +12,7 @@ enum BraveS3Resource: Hashable, DownloadResourceInterface {
   /// Generic iOS only content blocking behaviours used for the iOS content blocker
   case genericContentBlockingBehaviors
   /// Cosmetic filter rules
+  /// - Warning: Do not use this. This is here solely so we can delete the files
   case generalCosmeticFilters
```

Comment:
Additionally can you rename this like .deprecatedGeneralCosmeticFilters so name will suggest, it is not proper going to be used anymore.

Reply:
Done